src/app/controllers/censusdata/create.js

```diff
@@ -1,5 +1,24 @@
 'use strict';
 
+flnd.birdSpeciesList = {
+    retrieveRecords: function(config, $scope, messenger, authService) {
```

Comment:
I have not fully converted the project to this yet, but I've started making Service classes for CRUD actions, and then calling those classes from the controllers. In that paradigm, this method would move to a *Foraging*Service (this is for the foraging form, right? rather than the rookery census?).